rocket-okapi/src/request/from_form_param_impls.rs

```diff
@@ -74,6 +74,30 @@ impl<'r> OpenApiFromFormField<'r> for &'r str {
     }
 }
 
+#[cfg(feature = "uuid")]
+impl OpenApiFromFormField<'_> for rocket_contrib::uuid::Uuid {
+    fn form_parameter(gen: &mut OpenApiGenerator, name: String, required: bool) -> Result {
+        let schema = gen.json_schema::<uuid::Uuid>();
+        Ok(Parameter {
+            name,
+            location: "query".to_owned(),
+            description: None,
+            required,
+            deprecated: false,
+            allow_empty_value: false,
+            value: ParameterValue::Schema {
+                style: None,
+                explode: None,
+                allow_reserved: false,
+                schema,
+                example: None,
+                examples: None,
+            },
+            extensions: Default::default(),
```

Comment:
```suggestion
            extensions: okapi::Map::default(),
```

Please use explicit `default`